### DISCUSSION TITLE
Frequent crashes

### DISCUSSION THREAD

**Discussion by DocenciaLibre** [texstudio-org/texstudio#3244] [General]
Hello, I am using TeXstudio on Ubuntu 22.04. I have the TeXstudio PPA, and since it updated to version 4.6.2, there are frequent crashes. I have attached a screenshot of the crash and the backtrace file. Before this update, I never had this problem.

SIGSEGV (TXS-Version 0.0.0++ R )
[bt] texstudio(_ZN12SimulatedCPU13set_from_realEv+0x45) [0x557b8ce89ef5]
[bt] texstudio(_Z15print_backtraceRK7QString+0x46) [0x557b8ce89509]
[bt] texstudio(_ZN9Texstudio16recoverFromCrashEv+0x3cf) [0x557b8cd3f043]
[bt] texstudio(_Z7recoverv+0xd) [0x557b8cd3ec71]
[bt] texstudio(_ZNK14QNFADefinition11getPMatchesERK15QDocumentCursorR5QListINS_6PMatchEE+0x146) [0x557b8ce7cdbc]
[bt] texstudio(_ZN14QNFADefinition5matchER15QDocumentCursor+0x343) [0x557b8ce7ca01]
[bt] texstudio(_ZN7QEditor25emitCursorPositionChangedEv+0x118) [0x557b8cdce7c4]
[bt] texstudio(_ZN7QEditor9setCursorERK15QDocumentCursorb+0x255) [0x557b8cdcc68b]
[bt] texstudio(_ZN7QEditor15mousePressEventEP11QMouseEvent+0xa0f) [0x557b8cdd5bf3]
![image](https://github.com/texstudio-org/texstudio/assets/19748600/61f24c21-1cd3-4896-bd67-0b9d398a8402)

**Comment by sunderme** [comment]
I would need a test document and a description on how to provoke a crash.

**Reply by DocenciaLibre** [reply]
I will try, but the document is quite large, about 100 pages. I have reverted to version 4.2.1, which has no problem handling it. I will try to find a minimal version that causes the crash, but I'm not sure I can do it.

**Reply by sunderme** [reply]
You could send the document directly to me (sunderme@gmx.de)
Without some hints how to get a crash, there is little hope for a fix.

**Reply by DocenciaLibre** [reply]
The last update seems to have solved this issue, thank you very much!
